components/Image/image.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/header.css">
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="image.css">
+    <title>Image</title>
+</head>
+
+<body>
+    <header class="header fixed-header">
+        <div class="header-left-icons">
```

Comment:
Why is whole image HTML file in this PR, there is different branch for image and input

Reply:
actually this branch has some of the conflicts thats why my all the files are here i need to check it..btw thanks for suggestions i will definitely look into it